### ISSUE TITLE
Change or remove art which makes players uncomfortable.

### ISSUE THREAD

**Issue by dwvanstone** [buttonmen-dev/buttonmen#2481]
This came out of the forum:

http://buttonweavers.com/ui/forum.html#!threadId=765&postId=16083

"So if we're looking at things that make people uncomfortable, will previous Fanatics be edited?"
"Is there a particular Fanatic that you strongly feel shouldn't be here?"
"Gman97218"

I wanted to pull it out as a separate discussion. There may be other art on the site that makes people uncomfortable (or may turn potential players away from the site) beyond fanatics.

**Comment by AdmiralJota** [comment]
Also from that thread:

"As for offensive imagery, Updog displays the confederate battle flag."

(I assume he means [opedog](http://buttonweavers.com/ui/buttons.html?button=opedog).)

**Comment by AdmiralJota** [comment]
In any case, I'm not personally bothered by [gman97218](http://buttonweavers.com/ui/buttons.html?button=gman97216), but if many people are, I don't think it would be a big problem to put a little censor blur over the finger (keeping it within the circle, so it looks like part of the button).

**Comment by jl8e** [comment]
The only other button art I can think of that people might find offensive is the barely-concealed nipples  on the xxxenophile buttons. I do not think they should be changed -- they're actual physical buttons.

I don't think that we should change the gman art, either. It's more absurd than offensive. 

I do think we should alter the opedog art; the offense there is along a different axis. It's not like blurring it out will damage the point of the picture; the flag appears to be incidental. (And Georgia stopped using that flag early this century, for the same reason I think we should blur it.)

**Comment by AdmiralJota** [comment]
Ah, Georgia! That would explain it. It didn't quite look right for a Confederate flag, and it seemed really weird to have a photo of him standing there in front of a real Confederate flag waving, but I couldn't see what the explanation could have been.

I don't suppose anyone has the Photoshop skills to seamlessly replace it with the current flag :)

**Comment by blackshadowshade** [comment]
I just wanted to comment here that I'm personally not offended by very much at all, but I'm happy to act if there's a genuine issue here, as judged by some others who will understand the issues better than me (e.g., the whole confederate flag thing, which I have no idea about).

**Comment by jl8e** [comment]
> Ah, Georgia! That would explain it. It didn't quite look right for a Confederate flag, and it seemed really weird to have a photo of him standing there in front of a real Confederate flag waving, but I couldn't see what the explanation could have been.

Yeah, it looks like a vacation photo to me, with the flag either incidental to the backdrop or there (along with the sign) to establish location. 

> e.g., the whole confederate flag thing, which I have no idea about).

I can briefly explain in email if you'd like.

**Comment by blackshadowshade** [comment]
If it comes down to a situation where I need to understand to make a decision, then I'm happy to hear about the details, but otherwise, I'm happy to be guided by others in this.

**Comment by CarltonN** [comment]
I, like James, am rather difficult to offend. None of the images bother
me. That said, the issue here, i believe, is more about what new users
might find offensive. For the aforementioned reasons, i am probably not the
best to address those issues.

Carlton Noles
-------------------

**Comment by jl8e** [comment]
I thought of this thread when we were talking about flavor text changes. I think there's consensus on opedog, and not on the others. I'll make a separate issue for opedog.
